L/libblastrampoline/build_tarballs.jl

```diff
@@ -41,5 +41,5 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.9",
+               julia_compat="1.9", preferred_gcc_version = v"6"
```

Comment:
```suggestion
               julia_compat="1.10"
```

Reply:
I was thinking that we may want to backport to 1.9.

Reply:
Perhaps that is unlikely?